client/styles/importer.less

```diff
@@ -1,3 +1,5 @@
+@cache-version: `process.env.npm_package_version` ;
```

Comment:
That's just how it works. Its a default thing with node, all the npm stuff is there. Its not really an environment variable, its only available in node, but because this is run in node it works. Feel free to find a better way, but I doubt it.